docs/develop/01_sdk/01_quickstart.md

```diff
@@ -44,7 +79,7 @@ Inside the `kilt-rocks` project directory, install the **KILT SDK**, **Typescrip
 
 ```bash npm2yarn
 npm init -y
-npm install @kiltprotocol/sdk-js ts-node typescript axios
+npm install @kiltprotocol/sdk-js @kiltprotocol/did @kiltprotocol/credentials ts-node typescript axios
```

Comment:
the import of the credentials package is for the types, right? That's not ideal... we should really see that these are available from the types package or even sdk-js